### PR TITLE
update NEWS

### PR DESCRIPTION
**Pull request by donyunardi**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # goshawk 0.1.13.9009
 
+### Fixes
+* Fixed `h_caption_loqs_label` to handle multiple parameters when data frame is returned from `h_identify_loq_values`.
+
+### Enhancements
+* Implemented `nestcolor` in goshawk.
```
Comment (mhallal1) [inline NEWS.md:7]: Actually, there is a NEWS entry for `nestcolor` that was wrongly entered in the past release: https://github.com/insightsengineering/goshawk/pull/165/files

Please remove the wrong entry or replace the above with it.

File: NEWS.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+
+### Miscellaneous
```
Comment (mhallal1) [inline NEWS.md:9]:
```suggestion
### Miscellaneous

```

File: NEWS.md [hunk 3]
```diff
@@ -1,5 +1,14 @@
 # goshawk 0.1.13.9009
 
+### Fixes
+* Fixed `h_caption_loqs_label` to handle multiple parameters when data frame is returned from `h_identify_loq_values`.
+
+### Enhancements
```
Comment (mhallal1) [inline NEWS.md:6]:
```suggestion
### Enhancements

```

File: NEWS.md [hunk 4]
```diff
@@ -1,5 +1,14 @@
 # goshawk 0.1.13.9009
 
+### Fixes
```
Comment (mhallal1) [inline NEWS.md:3]:
```suggestion
### Fixes

```

File: NEWS.md (hunk 1 above)
Comment (pawelru) [reply NEWS.md:7]: This is not the first time I am seeing nestcolor related news entry inputted in the past release. May I ask you to double-check that for other packages as well?

Comment (donyunardi) [reply NEWS.md:7]: Thanks for catching this.
I'll replace the above with it.

**Review by mhallal1 — APPROVED** [review]

**Pull request merged**